de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/transfer/HUTransformService.java

```diff
@@ -346,9 +348,11 @@ public void cuToExistingTU(
 		}
 		else
 		{
+			// TODO implement getCreatedHUs in HUListAllocationSourceDestination or chnge this logic. Currently it throws an error here in care of aggregated hus
```

Comment:
what shall we do with this TODO?

Reply:
this is not needed any more (at least for the moment)  because of the if statement above it 
```
if (handlingUnitsBL.isAggregateHU(targetTuHU))
		{
			return Collections.emptyList(); // we are done; no attaching
		}

```